wagtail/admin/userbar.py

```diff
@@ -28,6 +28,22 @@ def render(self, request):
 class AccessibilityItem(BaseItem):
     """A userbar item that runs the accessibility checker."""
 
+    # Behaves as a static class attribute
+    _dialog_position = "right"
+
+    def __init__(self, dialog_position=None):
+        # Overwrite dialog_position class attribute if new value is provided
+        if dialog_position:
+            self.dialog_position = dialog_position
+
+    @property
+    def dialog_position(self):
+        return type(self)._dialog_position
+
+    @dialog_position.setter
+    def dialog_position(self, val):
+        type(self)._dialog_position = val
```

Comment:
I'm not sure we should tie the accessibility item implementation with the positioning. It adds a layouting concern in its signature, when it really should be the userbar's concern. I think we should try to keep the changes within the markup/CSS instead of manipulating the Python code here.